lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -65,15 +65,15 @@ export class SilentFlowClient extends BaseClient {
             if (this.config.serverTelemetryManager) {
                 this.config.serverTelemetryManager.incrementCacheHits();
             }
-            return await this.generateResultFromCacheRecord(cacheRecord, request.resourceRequestMethod, request.resourceRequestUri);
+            return await this.generateResultFromCacheRecord(cacheRecord, request.resourceRequestMethod, request.resourceRequestUri, request.clientClaims);
```

Comment:
Can we condense this to just pass request?

Reply:
Should I make it a `BaseAuthRequest` to cover every auth request type?

Reply:
Yeah that's fine for now

Reply:
Done!